frontend/sw360-portlet/src/main/webapp/html/search/view.jsp

```diff
@@ -44,7 +44,7 @@
             <tbody style="background-color: #f8f7f7; border: none;">
             <tr>
                 <td>
-                    <input type="text" maxlength="100" style="width: 200px; padding: 5px; color: gray;height:20px;"
```

Comment:
Yes, you are right. I copy/pasted too much. Changed it to include to maxlength again.